LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/EC/EmbeddedController.cs

```diff
@@ -209,10 +270,14 @@ private bool TryUpdateData()
                 embeddedControllerIO.Read(_registers, _data);
                 return true;
             }
-            catch (WindowsEmbeddedControllerIO.BusMutexLockingFailedException)
+            catch (IOException)
             {
                 return false;
             }
         }
+
+        public class IOException: ApplicationException {
```

Comment:
Any reason not to use the built-in IO exception?

Reply:
Probably no reason.